src/test/java/com/o19s/es/ltr/ranker/parser/XGBoostJsonParserTests.java

```diff
@@ -196,32 +275,6 @@ public void testComplexModel() throws Exception {
         }
     }
 
-    public void verifyModelWithParams(String params) throws IOException {
-        String model = params +
-                "[{" +
-                "\"nodeid\": 0," +
-                "\"split\":\"feat1\"," +
-                "\"depth\":0," +
-                "\"split_condition\":0.123," +
-                "\"yes\":1," +
-                "\"no\": 2," +
-                "\"missing\":2,"+
-                "\"children\": [" +
-                "   {\"nodeid\": 1, \"depth\": 1, \"leaf\": 0.5}," +
-                "   {\"nodeid\": 2, \"depth\": 1, \"leaf\": 0.2}" +
-                "]}]";
-
-        FeatureSet set = new StoredFeatureSet("set", singletonList(randomFeature("feat1")));
-        NaiveAdditiveDecisionTree tree = parser.parse(set, model);
-        FeatureVector v = tree.newFeatureVector(null);
-        v.setFeatureScore(0, 0.124F);
-        assertEquals(0.2F, tree.score(v), Math.ulp(0.2F));
-        v.setFeatureScore(0, 0.122F);
-        assertEquals(0.5F, tree.score(v), Math.ulp(0.5F));
-        v.setFeatureScore(0, 0.123F);
-        assertEquals(0.2F, tree.score(v), Math.ulp(0.2F));
-    }
-
     private float sigmoid(float f) {
```

Comment:
needed?

Reply:
It is used in one test, but I can just precompute the values.